### PR TITLE
Update "Create Position" Table UI

### PR DESCRIPTION
**Pull request by cryptotails**
## Issue ticket number & link
#648

### CODE REVIEW

**Comment by cryptotails** [comment]
## Before

Screenshot 2022-08-09 at 19 34 58

## After

File: src/components/antd/button-outline/index.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ interface Props extends ButtonProps {
   isActive?: boolean
   height?: 'lg' | undefined
   rounded?: boolean
+  square?: boolean
```
Comment (0xTARC) [inline src/components/antd/button-outline/index.tsx:10]: Doesn't really make sense to have a default, rounded, _and_ square variant

Is adding a square variant to this button necessary? Looks like it's only used in the custom pagination component you added in this PR. Could you just pass in same width and height on a css class instead? or am i missing something

File: pages/create-position/[collateralId]/open/index.tsx [hunk 2]
```diff
@@ -335,13 +335,15 @@ const OpenPosition = () => {
       value: collateral?.maturity ? parseDate(collateral?.maturity) : '--:--:--',
     },
     {
-      title: 'Face Value',
-      tooltip: 'The redeemable value of the bond at maturity.',
+      title: 'Value at Maturity',
+      tooltip:
+        'The redeemable price of the asset at maturity as determined by the current spot price of the underlier and the amount of collateral deposited.',
```
Comment (0xTARC) [inline pages/create-position/[collateralId]/open/index.tsx:340]: where did you get these info block text updates from?

File: pages/create-position/index.tsx [hunk 3]
```diff
@@ -71,7 +71,7 @@ const CreatePosition = () => {
           value={`$${getHumanValue(value ?? 0, WAD_DECIMALS)?.toFixed(2)}`}
         />
       ),
-      title: 'Face Value',
+      title: 'Price At Maturity',
```
Comment (0xTARC) [inline pages/create-position/index.tsx:74]: again where did these text updates come from? could be correct, i just don't know what's preferred

File: src/components/antd/button-outline/index.tsx (hunk 1 above)
Comment (cryptotails) [reply src/components/antd/button-outline/index.tsx:10]: It's true that the square variant is only used here. Passing in a css class sounds good!

File: pages/create-position/[collateralId]/open/index.tsx (hunk 2 above)
Comment (cryptotails) [reply pages/create-position/[collateralId]/open/index.tsx:340]: These must have come from the merging the latest updates on dapp-redesign back into this branch  - I didn't update this

File: pages/create-position/index.tsx (hunk 3 above)
Comment (cryptotails) [reply pages/create-position/index.tsx:74]: Same here

File: src/components/antd/button-outline/index.tsx (hunk 1 above)
Comment (cryptotails) [reply src/components/antd/button-outline/index.tsx:10]: Do you have any ideas for overcoming that?

Comment (0xTARC) [reply src/components/antd/button-outline/index.tsx:10]: took a stab at this for a while, it was quite spicy. ideally we use antd button with the `ghost` prop, but that's a bigger tech debt lift we don't need to worry about for now

will go ahead and merge

**Review by 0xTARC — APPROVED** [review]

**Pull request merged**